styles/styles.css

```
@@ -572,3 +663,16 @@ h3 {
 .page-footer{
     background-color: #FFFFFF;
 }
+
+.page-footer-container {
+    display: flex;
+    width: 1200px;
+    margin: 0 auto;
+    justify-content: space-around;
+}
+
+.advantages-list {
+    display: flex;
+    justify-content: center;
+
+}
```

```suggestion
.advantages-list {
    display: flex;
}
```
Тут не нужен center, ведь если элементов будет не кратно 3, то центрирование будет выглядеть нелепо.